pkg/kuttlctl/cmd/test.go

```diff
@@ -161,6 +162,10 @@ For more detailed documentation, visit: https://kudo.dev/docs/testing`,
 				options.ArtifactsDir = artifactsDir
 			}
 
+			if isSet(flags, "namespace") {
+				options.Namespace = namespace
```

Comment:
An interesting corner case is what happens when user specifies `--namespace ''` (this can happen intentionally or due to a misspelled variable name in a script for example).
Perhaps we should validate that the flag value is non-empty if specified explicitly?

Reply:
nice... agree!   --namespace="" explicit should mean cluster scope which we do not allow and should explicit check for... thanks @porridge